scripts/deploy-assets.sh

```diff
@@ -0,0 +1,8 @@
+# !/usr/bin/bash
+
+NODE_ENV=production mkdir public; \
+mkdir public/assets; \
+ezel-assets mobile/assets/ && \
+ezel-assets desktop/assets/ && \
+bucket-assets --bucket artsy-force-$DEPLOY_ENV && \
```

Comment:
You can also start  your script with `set -e` so you don’t need to add all the `&& \` parts.

```
sh-3.2$ help set
set: set [--abefhkmnptuvxBCHP] [-o option] [arg ...]
        -e  Exit immediately if a command exits with a non-zero status.
        -x  Print commands and their arguments as they are executed.
```

May also want to set that second one.